docs/zh/UserGuide/Maintenance-Tools/Maintenance-Command.md

```diff
@@ -175,6 +175,28 @@ IoTDB> show schema regions
 Total line number = 2
 It costs 0.012s
 ```
+## 集群 DataNode 分布式监控工具
+
+当前 IoTDB 支持使用如下 SQL 查看 DataNode：
+
+- `SHOW DATANODES`: 展示所有DataNode
+
+```sql
+IoTDB> create timeseries root.sg.d1.s1 with datatype=BOOLEAN,encoding=PLAIN
+Msg: The statement is executed successfully.
+IoTDB> create timeseries root.sg.d2.s1 with datatype=BOOLEAN,encoding=PLAIN
+Msg: The statement is executed successfully.
+IoTDB> create timeseries root.ln.d1.s1 with datatype=BOOLEAN,encoding=PLAIN
+Msg: The statement is executed successfully.
+IoTDB> show regions
++--------+------------+------+-------------+-----+----------+---------+----+
+|RegionId|        Type|Status|storage group|Slots|DataNodeId|     Host|Port|
++--------+------------+------+-------------+-----+----------+---------+----+
+|       0|SchemaRegion|    Up|      root.sg|    2|         2|127.0.0.1|6668|
+|       1|SchemaRegion|    Up|      root.ln|    1|         1|127.0.0.1|6667|
++--------+------------+------+-------------+-----+----------+---------+----+
+Total line number = 2
+It costs 0.035s
```

Comment:
Is the description and sql consistent?

Reply:
This failure occurs when fixing conflicts, fixed~